app/assets/stylesheets/core/_core.scss

```diff
@@ -1,205 +1,723 @@
-@import 'markus';
+// Core page styling
 
-@import 'navigation';
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 
+html,
+body {
+  min-height: 100%;
+  width: 100%;
+}
 
-#about_dialog {
-  width: 50%;
-  max-height: 75%;
+body {
+  background-color: $lighter-blue;
+  color: $dark-grey;
+  font: 400 0.8em/1.2em $fonts;
```

Comment:
`0.8em/1.2em` should be written with a single space on each side of the operator: `0.8em / 1.2em`